cli/types/cypress.d.ts

```diff
@@ -2609,6 +2609,109 @@ declare namespace Cypress {
     includeShadowDom: boolean
   }
 
+  /**
+   * Options appended to config object on runtime.
+   */
+  interface RuntimeConfigOptions {
+    /**
+     * CPU architecture, from Node `os.arch()`
+     *
+     * @see https://nodejs.org/api/os.html#os_os_arch
+     */
+    arch: string
+    /**
+     * The list of hosts to be blocked
+     */
+    blockHosts: null | string | string[]
+    /**
+     * The browser Cypress is runnong on.
+     */
+    browser: Browser
+    /**
+     * The browsers Cypress can use.
+     */
+    browsers: Browser[]
+    /**
+     * Path to folder containing component test files.
+     */
+    componentFolder: string
+    /**
+     * Enables component testing.
+     */
+    experimentalComponentTesting: boolean
+    /**
+     * Enables polyfill fetch.
+     */
+    experimentalFetchPolyfill: boolean
+    /**
+     * ???
+     */
+    hosts: null | string[]
```

Comment:
I think this is also mostly used internally? We set it in our Cypress config, but I"m not sure why someone else would need this.

Reply:
well, TIL. 

```suggestion
    /**
     * Hosts mappings to IP addresses.
     */
    hosts: null | string[]
```